### ISSUE TITLE
Glycosylation summary issue

### ISSUE THREAD

**Issue by jeet-vora** [glygener/glygen-issues#1628]
There is a discrepancy in the glycosylation summary.

`UAC - Q08048`

On the portal and the API it says there are **`4 site(s) total, 6 N-linked site(s)`**
However, there are only 4 N-linked sites in total instead of 6 and 2 N-linked Glycans in total. Please check

https://glygen.org/protein/Q08048/#Glycosylation

<img width="885" alt="image" src="https://github.com/user-attachments/assets/ec5a81de-1021-4d50-b420-78b78554ce9b">

https://api.glygen.org/protein/detail/Q08048/
<img width="351" alt="image" src="https://github.com/user-attachments/assets/2ec18a55-cc8f-4490-a16d-34d46abf553d">

**Comment by rykahsay** [comment]
There are 4 unique "protein positions" and 6 unique "protein position + glycan" combinations, all of them being type N-linked. The summary text should be changed to clear the confusion

```
295 G80920RR
656 G71009JJ
295 uknown
656 uknown
403 uknown
569 uknown
```

**Comment by katewarner** [comment]
`4 protein positions, 2 glycosylated by known glycans, and 4 glycosylated by unknown glycans `
`4 protein positions, 2 glycosylated by 2 known and 2 unknown glycans, 2 glycosylated by 2 unknown glycans`

**Comment by jeet-vora** [comment]
Has this issue resolved @katewarner

**Comment by katewarner** [comment]
@jeet-vora  No, I'll move it to 2.8 and add it to the next  developer meeting agenda

**Comment by katewarner** [comment]
@rykahsay apologies I forgot to assign this to you. I think we discussed this during 2.7 and decided on the summary in this comment: https://github.com/glygener/glygen-issues/issues/1628#issuecomment-2297026098

Please implement for 2.8

**Comment by rykahsay** [comment]
This is the same issue as the one we discussed to day -- Rene is going to have a look at the other ticket
